app/components/SidebarLayout.tsx

```diff
@@ -170,6 +171,13 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
                 label="Users"
                 href="/users"
               />
+              <SideBarListItem
+                isCollapsed={isCollapsed}
+                isCurrentPage={segments.includes("fatalities")}
+                Icon={FaHeart}
```

Comment:
I see the heart as rendered with a little tiny bit of it cut off on the right. I have toyed with this SideBarListItem component trying different utility classes to try to open it up a little more, but no dice. I'm open to any help on this one.

<img width="493" alt="image" src="https://github.com/user-attachments/assets/dfce247a-5417-4350-9b27-769b6f966f62" />